docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -140,20 +159,31 @@ whenCreated                        : 3/16/2009 4:15:00 PM
 This command gets a specific computer showing all the properties.
 
 ### Example 2: Get all computers with a name starting with a particular string
+
 ```powershell
-PS C:\> Get-ADComputer -Filter 'Name -like "User01*"' -Properties IPv4Address | FT Name,DNSHostName,IPv4Address -A
+Get-ADComputer -Filter 'Name -like "User01*"' -Properties IPv4Address | 
+    Format-Table Name,DNSHostName,IPv4Address -AutoSize
```

Comment:
Where possible, we want to have spaces between list items so they're easier to read.

```suggestion
    Format-Table Name, DNSHostName, IPv4Address -AutoSize
```